modules/standard/raid/auction_controller.py

```diff
@@ -33,7 +33,7 @@ def auction_start_cmd(self, request, _, items):
         if self.is_auction_running():
             return "Auction already running."
 
-        items = re.findall(r"(([^<]+)?<a href=\"itemref://(\d+)/(\d+)/(\d+)\">([^<]+)</a>([^<]+)?)", items)
+        items = re.findall(r"(([^<]+)?<a href=[\"\']itemref://(\d+)/(\d+)/(\d+)[\"\']>([^<]+)</a>([^<]+)?)", items)
```

Comment:
there might be a few other places to make this change as well.  this is one of them: https://github.com/Budabot/Tyrbot/blob/master/core/command_param_types.py#L225 but I think there might be a few other also